src/stores/courseData.js

```diff
@@ -171,7 +171,10 @@ const store = new Vuex.Store({
       }
     },
     removeClass (state, classInfo) {
+      console.log(classInfo);
+      console.log(state.roads[state.activeRoad].contents.selectedSubjects);
       const classIndex = state.roads[state.activeRoad].contents.selectedSubjects.indexOf(classInfo);
+      console.log(classIndex);
```

Comment:
FYI you probably want to remove this before merging master in, it caused some conflicts when I did that locally. (and ofc before merging the PR too)